src/mozillavpn.h

```diff
@@ -439,6 +443,7 @@ class MozillaVPN final : public QObject {
   QTimer m_alertTimer;
   QTimer m_periodicOperationsTimer;
   QTimer m_gleanTimer;
+  QTimer m_serverCooldownTimer;
```

Comment:
Where do we use this?

Reply:
Thank you! I forgot to remove that while refactoring.